supplierinfo_product_breakdown/tests/test_supplierinfo.py

```diff
@@ -0,0 +1,40 @@
+from odoo.tests import tagged
+
+from .common import SupplierInfoCommon
+
+
+@tagged("post_install", "-at_install")
```

Comment:
Please provide test scenario. What are you trying to test exactly?

Reply:
Resolved